ui-admin/src/study/participants/datarepo/DatasetDashboard.tsx

```diff
@@ -117,9 +117,10 @@ const DatasetDashboard = ({ studyEnvContext }: {studyEnvContext: StudyEnvContext
                   <br/>
                   <label>Dataset ID:</label> { datasetDetails?.datasetId }
                   <br/>
-                  <label>Created Date:</label> { instantToDefaultString(datasetDetails?.createdAt) }
+                  <label>Date Created:</label> { instantToDefaultString(datasetDetails?.createdAt) }
                   <br/>
-                  <label>Last Successful Export:</label> { instantToDefaultString(datasetDetails?.lastExported) }
+                  <label>Description:</label> { datasetDetails?.description ?
+                    datasetDetails?.description : <em>N/A</em> }
```

Comment:
Nit: I think I'd replace the `<em>` with a `<span className="fst-italic">`. `em` is for emphasizing text, and I'm not sure that really fits here.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em

Reply:
Done 725468b77eacb448f4c198fc3e6debfd55364422